core/src/lib/jcr.cc

```diff
@@ -1013,12 +1013,10 @@ void DbgPrintJcr(FILE* fp)
   fprintf(fp, "Attempt to dump current JCRs. njcrs=%d\n",
           job_control_record_chain->size());
 
+  std::size_t num_dumped = 0;
   for (JobControlRecord* jcr
        = (JobControlRecord*)job_control_record_chain->first();
        jcr; jcr = (JobControlRecord*)job_control_record_chain->next(jcr)) {
-    fprintf(fp, "threadid=%s JobId=%d JobStatus=%c jcr=%p name=%s\n",
-            edit_pthread(jcr->my_thread_id, ed1, sizeof(ed1)), (int)jcr->JobId,
-            jcr->getJobStatus(), jcr, jcr->Job);
```

Comment:
did we actually dump that information twice previously?

Reply:
What do you mean ? The same information is also included in the next printf.
I also saw backtraces were this was indeed the case.  Do you have any without this message twice ?

Reply:
that's why I asked...

Reply:
ah good. i thought i missed something obvious again.